core/src/plugins/filed/python/module/bareosfd.cc

```diff
@@ -2168,12 +2160,13 @@ static int PyIoPacket_init(PyIoPacket* self, PyObject* args, PyObject* kwds)
   self->whence = 0;
   self->offset = 0;
   self->win32 = false;
+  self->filedes = -1;
```

Comment:
that should probably use the constant, too.
```suggestion
  self->filedes = kInvalidFiledescriptor;
```